CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
+
+Welcome to systemd source-git for CentOS Stream and RHEL. When contributing, please follow the guide below.
+
+## Workflow
+
+```mermaid
+flowchart LR
+    A(Issue) --> B{is fixed\nupstream}
+    B -->|YES| C(backport\nupstream patch)
+    B -->|NO| D(upstream\nsubmit issue or PR)
+    D --> E{accepted\nand fixed}
+    E -->|YES| C
+    E -->|NO| F(rhel-only patch) --> G
+    C --> G(submit PR)
+```
+
+## Filing issues
+
+When you find an issue with systemd used in CentOS Stream or RHEL, please file an issue by using Jira ticket system - [link](https://issues.redhat.com/secure/CreateIssue!default.jspa)
```

Comment:
I'd formulate this paragraph as follows:

When you find an issue with systemd used in CentOS Stream or RHEL, please file an issue in [Jira ticket system](https://issues.redhat.com/secure/CreateIssue!default.jspa) (set Project to **RHEL** and Component to **systemd**).

But that's mostly a matter of preference...

Reply:
In any case, the link can be set on the words "Jira ticket system".

Reply:
But I like your preference :) :+1:

Reply:
Done. Thanks for the feedback.